.github/workflows/test.yml

```
@@ -18,13 +20,15 @@ jobs:
           distribution: 'corretto'
 
       - name: Configure AWS credentials
+        if: github.event_name == 'push'
         uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)